criterion-compare.cabal

```
@@ -19,12 +19,12 @@ executable criterion-compare
   hs-source-dirs:      src
   other-extensions:    GeneralizedNewtypeDeriving, RecordWildCards, OverloadedStrings, FlexibleContexts, TupleSections
   default-language:    Haskell2010
-  build-depends:       base >=4.9 && <= 4.13,
+  build-depends:       base >=4.9 && <= 4.14,
```

I noticed that I can already build cleanly with 8.10.1-alpha2…

```suggestion
  build-depends:       base >=4.9 && <= 4.150,
```

---

Don't use `<=` (less than or **equal to**). It's definitely not right, it should be `< 4.15` (less than)

---

Oops! ;)